variables-and-memory/problem-set-variables-and-memory.md

```diff
@@ -78,20 +78,20 @@ The variable `result` on line `7` is defined in function scope. The variable `re
 What is the value of `result` after the function `is_letter_in_word` is invoked as shown below?
 
 ```py
-1  def is_letter_in_word(word, letter):
-2       """
-3       input: word (string) and letter (single character)
-4       output: boolean
-5       """
-6       if letter not in word:
-7           result = False
-8       else: 
-9           result = True
-10
-11      return result
-12
-13 result = True
-14 result = is_letter_in_word("hello", "x"):
```

Comment:
This one too. If the line numbers are going to be removed, we should reword the explanation.

Reply:
oof, I thought this is why I had the numbers, but then when I reread it, I didn't see it!

Reply:
The nice thing about removing the line numbers is that it allows the code to be copied and pasted into an editor more easily. Learn isn't smart about not selecting them otherwise. So I'm in favor of removing them. But yeah, the explanation.

Reply:
I'm going to leave the line numbers for now, and make an asana card to change this